sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/component.py

```diff
@@ -347,6 +347,7 @@ def _from_container_rest_object(cls, component_container_rest_object: ComponentC
             # Set this field to None as it hold a default True in init.
             is_deterministic=None,
         )
+        component.latest_version = component_container_details.latest_version
```

Comment:
component_container_details

Reply:
I think we shouldn't add properties here? The `ComponentContainerDetails` already has `latest_version` attribtue. 
![image](https://user-images.githubusercontent.com/49266998/203259320-a0d9495e-4a58-428e-ab38-daf417099e51.png)